test/test_model.py

```diff
@@ -403,15 +403,15 @@ def generalTest(self, _model, admin, user):
         # Test with fields
         for currentUser in (user, admin):
             query, fields = _model._textSearchFilters('names')
-            assert 'name' in _model.findWithPermissions(
-                query, fields=fields, user=currentUser).next()
-            assert 'field1' in _model.findWithPermissions(
-                query, fields=fields, user=currentUser).next()
+            assert 'name' in next(_model.findWithPermissions(
```

Comment:
This is wrong.  `next()` is a mongo cursor call, not a Python iterator built-in.

Reply:
After discussion with @manthey in-person, we determined that a `pymongo.Cursor` or a `pymongo.CommandCursor` [is a Python iterator](https://github.com/mongodb/mongo-python-driver/blob/master/pymongo/cursor.py#L1162), as it implements the Python iterator protocol (`next` / `__next__` and `__iter__`), so we can advance it in a generalized way with `next()`.

This should be good to stay as-is.